README.md

```diff
@@ -25,24 +27,20 @@ with:
 
 **Required** The path the root directory of your static website or other content that you want to publish to IPFS.
 
-### `web3_token`
-
-**Required** API token for web3.storage
+### `secret_key`
 
-<details>
-  <summary>Show advanced options: <code>wrap_with_directory</code>, <code>include_hidden</code>, <code>web3_api</code></summary>
+**Required** The base64 key to use to sign ucan invocations to web3.storage. 
 
-### `wrap_with_directory`
-
-_Default_ `false`
+Create one using `w3 key create`. See: https://github.com/web3-storage/w3cli#w3_principal
 
-Should the `path_to_add` be wrapped in a diretory when creating the IPFS DAG. For most folks using this action the default of `false` is fine. 
+### `proof`
 
-This is the opposite of the default that web3.storage uses, as this action is commonly used to add a directory that contains a static website to IPFS. In that case you want the path_to_add to become the root cid so you can host your site at `https://<cid>.ipfs.dweb.link` rather than `https://<cid>.ipfs.dweb.link/<path_to_add>`.
+**Required** A base64 encoded UCAN delegating capabilities the signing key above. 
 
-If you do want to capture the `path_to_add` path itself in the IPFS DAG then you want to set `wrap_with_directory:true`.
+Create using `w3 delegation create --base64`
```

Comment:
should we mention in the docs that delegation can be narrowed to a set of capabilities and they should make it only for their needs pointing to the docs https://github.com/web3-storage/w3cli?tab=readme-ov-file#w3-delegation-create-audience-did ?

Reply:
yeah it is good practice for us to recommend restricting caps.